bin/mail/2017-teacher-application-responses/test_teacher_application_decision_processor.rb

```diff
@@ -89,7 +90,7 @@ def test_process_decisions_row_accept_teachercon
       name: 'Tracy Teacher',
       email: 'tracy.teacher@example.net',
       preferred_first_name_s: 'Tracy',
-      course_name_s: 'CS Principles',
+      course_name_s: 'CS Discoveries',
```

Comment:
This change is just to make the test more readable. This field isn't used here, but it was confusing me that it said the wrong course name.